website/blog/2022-01-06-prettier-begins-paying-maintainers.md

```diff
@@ -0,0 +1,22 @@
+---
+author: "Christopher Chedeau (@vjeux)"
```

Comment:
@vjeux  This article was originally written by me, but most have been rewritten by you. If it is inconvenient for your name to appear here, we can change it to me.

Reply:
Happy to have it under my name :)